schemas/Metals/v0.0.1/schema.json

```diff
@@ -0,0 +1,1287 @@
+{
+  "meta:license": ["Copyright 2025 S1Seven Gmbh. All rights reserved."],
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "$id": "https://schemas.materialidentity.org/metals-schemas/v0.0.1/schema.json",
```

Comment:
Is this cname currently active? I remember we talked about it but I'm not sure if it was fully implemented.

Reply:
It is configured on Cloudflare (dzxs4jq1j6192.cloudfront.net), but something needs to be done on CloudFront. I remember talking to Gianni about it quite some time ago.